examples/step-71/step-71.cc

```diff
@@ -1472,20 +1554,20 @@ namespace Step71
       Tensor<3, dim, SD::Expression>          PP_SD;
       SymmetricTensor<4, dim, SD::Expression> HH_SD;
 
-      // This is the optimizer that is used to evaluate the dependent functions.
-      // More specifically, it provides possibility to accelerate the evaluation
+      // The next variable is then the optimizer that is used to evaluate the dependent functions.
+      // More specifically, it provides the possibility to accelerate the evaluation
       // of the symbolic dependent expressions. This is a vital tool, because
-      // the native evaluation of lengthy expressions can be very slow.
+      // the naive evaluation of lengthy expressions can be very slow.
       // The Differentiation::SD::BatchOptimizer class provides a mechanism by
       // which to transform the symbolic expression tree into another code path
       // that, for example, shares intermediate results between the various
       // dependent expressions (meaning that these intermediate values only
       // get calculated once per evaluation) and/or compiling the code using
-      // a just-in-time compiler (thereby retreiving near-native performance
+      // a just-in-time compiler (thereby retrieving near-native performance
       // for the evaluation step). 
       //
       // Performing this code transformation is very computationally expensive,
-      // so we store the optimizer so that its done just once per class instance.
+      // so we store the optimizer so that it's done just once per class instance.
```

Comment:
```suggestion
      // so we store the optimizer so that it's done just once per class instance.
```
```suggestion
      // so we store the optimizer so that it is done just once per class instance.
```